simple/testdata/sort.go

```diff
@@ -0,0 +1,14 @@
+package pkg
+
+import "sort"
+
+func fn() {
+	var a []int
+	sort.Sort(sort.IntSlice(a)) // MATCH "sort.Ints"
+
+	var b []float64
+	sort.Sort(sort.Float64Slice(b)) // MATCH "sort.Float64s"
+
+	var c []string
+	sort.Sort(sort.StringSlice(c)) // MATCH "sort.Strings"
+}
```

Comment:
Test cases for checks should include examples of code that mustn't be flagged. I'm thinking of the following ones:

1. sort.Sort with an arbitrary argument that is not sort.IntSlice/sort.Float64Slice/sort.StringSlice

Reply:
Another good test case to add, just to be safe: `sort.Sort(x)` where x is a named type with underlying type `[]int`